lib/BackgroundJob/TrainIpV4OnceJob.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\SuspiciousLogin\BackgroundJob;
+
+use OCA\SuspiciousLogin\Exception\InsufficientDataException;
+use OCA\SuspiciousLogin\Service\Ipv4Strategy;
+use OCA\SuspiciousLogin\Service\TrainingDataConfig;
+use OCA\SuspiciousLogin\Service\TrainService;
+use OCP\AppFramework\Utility\ITimeFactory;
+use OCP\BackgroundJob\Job;
+use Psr\Log\LoggerInterface;
+use Throwable;
+
+class TrainIpV4OnceJob extends Job {
```

Comment:
This should be QueuedJob, else the job won't be removed after execution. Fixing and testing right now.